components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -320,20 +320,27 @@ function matchFunctions.getOpponents(args)
 		local placement = 1
 		local lastScore
 		local lastPlacement = 1
+		local lastStatus
 		-- luacheck: push ignore
 		for opponentIndex, opponent in Table.iter.spairs(opponents, p._placementSortFunction) do
+			if opponent.status ~= 'S' and opponent.status ~= 'W' and placement == 1 then
```

Comment:
```suggestion
			if opponent.status ~= _STATUS_SOMETHING and opponent.status ~= _STATUS_SOMETHING_ELSE and placement == 1 then
```
No idea what these mean so would really help
No idea

Reply:
adjusted it there and everywhere else in this module